test/minimal/compare/output11-.env

```diff
@@ -0,0 +1,4 @@
+/*eslint-disable*/
```

Comment:
I actually got an eslint error here: "Use the global form of "use strict".", but this is actually not a js file